lua/nvim-lsp-installer/ui/display.lua

```diff
@@ -99,42 +99,72 @@ end
 
 function M.new_view_only_win(name)
     local namespace = vim.api.nvim_create_namespace(("lsp_installer_%s"):format(name))
-    local buf, renderer, mutate_state, get_state, unsubscribe
+    local buf_handle, renderer, mutate_state, get_state, unsubscribe, win_id
```

Comment:
Nitpick, but maybe `bufnr` and `winnr` for consistency?

Reply:
Funny you should ask, I was actually going for `bufnr` but I thought you specifically avoided them. I had to look it up because I could've sworn it was always `bufnr` in every instance I saw, but no! The official docs actually refers to them as a `handle` rather than a number. So it's a toss up for me, I'll let you decide.

Reply:
Hehe I remember using `bufnr`, `winnr` at first but later removed the `nr` because I ran into some quirks that suggested there's not a perfect 1:1 match between "Vim" and "Nvim Lua" buffer APIs, but I probably just did something wrong and came to a wrong conclusion, I'm still a bit new to these APIs.

I say if the "handle" terminology is the neovim standard, let's go with `buf_handle`, `win_handle` - otherwise `bufnr`, `winnr`. Again, nitpicking 😅

Reply:
If we're going by the general usage then it's `bufnr` and `win_id`, at least from what I've seen in core and Telescope.